test/libsolidity/syntaxTests/parsing/location_specifiers_for_locals.sol

```diff
@@ -1,11 +1,10 @@
 contract Foo {
-    function f() public {
-        uint[] storage x;
+    uint[] m_x;
+    function f() public view {
+        uint[] storage x = m_x;
         uint[] memory y;
     }
 }
 // ----
-// Warning: (49-65): Uninitialized storage pointer.
-// Warning: (49-65): Unused local variable.
-// Warning: (75-90): Unused local variable.
-// Warning: (19-97): Function state mutability can be restricted to pure
+// Warning: (70-86): Unused local variable.
```

Comment:
Perhaps these warnings can be silenced. I find them distracting since their not relevant in the scope of this test.